### PR TITLE
Comply with REUSE version 3.0 for licensing declaration.

### PR DESCRIPTION
**Pull request by apteryks**

### CODE REVIEW

File: library-files-utilities/srfi/%3A%104/library-files-utilities.r6rs-lib [hunk 1]
```diff
@@ -1,28 +1,8 @@
...
-;; THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-;; IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-;; FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
-;; THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-;; LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-;; FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
-;; DEALINGS IN THE SOFTWARE.
+;;;; SPDX-FileCopyrightText: 2010 Derick Eddington
+;;;;
+;;;; SPDX-License-Identifier: MIT
```
Comment (arthurgleckler) [inline library-files-utilities/srfi/%3A%104/library-files-utilities.r6rs-lib]: Alas, this has the same issue as SRFI 103.  It has the same author, too.

**Review by arthurgleckler — CHANGES_REQUESTED** [review]

File: library-files-utilities/srfi/%3A%104/library-files-utilities.r6rs-lib (hunk 1 above)
Comment (apteryks) [reply library-files-utilities/srfi/%3A%104/library-files-utilities.r6rs-lib]: We'll have to wait for their answer. If they do not reply I'll have to extract the license text and tag it with a custom SPDX license tag.

Comment (apteryks) [reply library-files-utilities/srfi/%3A%104/library-files-utilities.r6rs-lib]: Derick has been kind enough to reply by email to you and I that they accept to change the license from their MIT variant to plain MIT (which this change is doing); so this should now be fine!

**Comment by arthurgleckler** [comment]
Thanks once more.  Merged and pushed.

**Pull request merged**